Makefile

```diff
@@ -49,6 +49,7 @@ docker/weaveutil:
 $(SCOPE_EXPORT): $(SCOPE_EXE) $(DOCKER_DISTRIB) docker/weave docker/weaveutil $(RUNSVINIT) docker/Dockerfile docker/demo.json docker/run-app docker/run-probe docker/entrypoint.sh
 	cp $(SCOPE_EXE) $(RUNSVINIT) docker/
 	cp $(DOCKER_DISTRIB) docker/docker.tgz
+	cd docker && tar -xvzf docker.tgz
```

Comment:
this extracts every file from the tarfile; it would be better to just extract one 

    cd docker && tar -xvzf docker.tgz docker/docker

or skip the extra subdirectory and amend Dockerfile to suit:

    tar -xvzf docker/docker.tgz docker/docker